olp-cpp-sdk-core/include/olp/core/logging/Log.h

```diff
@@ -223,26 +225,26 @@
  *
  * @param tag Log component name.
  */
-#define LOG_FATAL_F(tag, ...) \
-  LOG_CRITICAL_F(::olp::logging::Level::Fatal, tag, __VA_ARGS__)
+#define EDGE_SDK_LOG_FATAL_F(tag, ...) \
+  EDGE_SDK_LOG_CRITICAL_F(::olp::logging::Level::Fatal, tag, __VA_ARGS__)
 
 /**
  * @brief Log a critical fatal error message using printf style formatting, then
  * abort the program.
  * @param tag Log component name.
  */
-#define LOG_ABORT_F(tag, ...)      \
-  do {                             \
-    LOG_FATAL_F(tag, __VA_ARGS__); \
-    std::abort();                  \
-  }                                \
-  CORE_LOOP_ONCE()
+#define EDGE_SDK_LOG_ABORT_F(tag, ...)      \
+  do {                                      \
+    EDGE_SDK_LOG_FATAL_F(tag, __VA_ARGS__); \
+    std::abort();                           \
+  }                                         \
+  EDGE_SDK_CORE_LOOP_ONCE()
 
 #ifdef LOGGING_DISABLED
-#define LOG(level, tag, message) \
-  do {                           \
-  }                              \
-  CORE_LOOP_ONCE()
+#define EDGE_SDK_LOG(level, tag, message) \
```

Comment:
I wonder why we don't need to mark the parameters as unused same as for:
```cpp
#ifdef LOGGING_DISABLE_DEBUG_LEVEL
#define LOG_TRACE(tag, message)           \
  do {                                    \
    ::olp::logging::NullLogStream __strm; \
    __strm << message;                    \
    CORE_UNUSED(tag, __strm);             \
  }                                       \
  CORE_LOOP_ONCE()
```

Reply:
Not a scope of the changes, but seems CORE_UNUSED is redundant in macro definition.